package.json

```diff
@@ -22,7 +22,9 @@
     "url": "git@github.com:cisco-ce/jsxapi.git"
   },
   "scripts": {
-    "build": "npm run build:js",
+    "build": "run-s build:js build:**",
+    "build:dist:min": "parcel build --global jsxapi -o jsxapi.min.js ./src/browser.ts",
+    "build:dist": "parcel build --no-minify --global jsxapi -o jsxapi.js ./src/browser.ts",
```

Comment:
Does `"build:dist": "build:dist:min --no-minify",` work?

Reply:
I don't think it'll work like that, but perhaps with `npm run ...` in front.